py/bitbox02/bitbox02/bitbox02/bitbox02.py

```diff
@@ -776,6 +776,14 @@ def eth_sign_msg(
         Signs message, the msg will be prefixed with "\x19Ethereum message\n" + len(msg) in the
         hardware
```

Comment:
Can you also add a comment here like in https://github.com/digitalbitbox/bitbox02-api-go/pull/55 for public docs:

> 27 is added to the recID to denote an uncompressed pubkey.

Reply:
done